cscs-checks/mch/automatic_arrays.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+import reframe.utility.sanity as sn
+from reframe.core.pipeline import RegressionTest
+
+
+class AutomaticArraysCheck(RegressionTest):
+    def __init__(self, **kwargs):
+        super().__init__('automatic_arrays_check',
+                         os.path.dirname(__file__), **kwargs)
```

Comment:
Can you use the new syntax for regression tests? This boilerplate code won't be needed any more, as well as the `_get_checks()` function.

Reply:
@ajocksch You can check the tutorial examples to see the actual syntax.

Reply:
done